dolphin/api/v1/storages.py

```diff
@@ -27,7 +52,70 @@ def show(self, req, id):
         return dict(name="Storage 2")
 
     def create(self, req, body):
-        return dict(name="Storage 3")
+        """
+        This function for registering the new storage device
+        :param req:
+        :param body: "It contains the all input parameters"
+        :return:
+        """
+        # Check if body is valid
+        if not self.is_valid_body(body, 'storages'):
```

Comment:
hope this is different from our API spec 
{
  "name": "EMC-VMAX-123456",
  "description": "VMAX storage lab1",
  "manufacturer": "EMC",
  "model": "VMAX",
  "hostName": "hostNAme",
  "userName": "admin",
  "password": "string",
  "extendedConnectionParams": {
    "controller1": "string",
    "ip1": "string",
    "shhKeyPath": "string"
  }
}